nilearn/plotting/surface/tests/test_surf_plotting.py

```diff
@@ -1088,19 +1115,24 @@ def test_plot_img_on_surf_with_engine_kwarg(img_3d_mni):
 
 
 def test_plot_img_on_surf_title(matplotlib_pyplot, img_3d_mni):
+    """Test nilearn.plotting.surface.plot_img_on_surf with and without title
+    specified.
+    .
+    """
     title = "Title"
-    fig, _ = plot_img_on_surf(
```

Comment:
I've changed the return value of `plot_img_on_surf` to return only `fig` instead of `fig, axes` to make it consistent with the other plotting functions. I couldn't find any example or that that uses axes. 

Would that be ok?

Reply:
- relates to https://github.com/nilearn/nilearn/issues/2761

I think that this is a change in behavior so would need to go through a deprecation cycle

So maybe better in a follow up PR?